cmd/testing/tls_test.go

```diff
@@ -0,0 +1,44 @@
+package cmdtesting
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/suite"
+)
+
+func TestTLSSuite(t *testing.T) {
+	suite.Run(t, new(TLSSuite))
+}
+
+type TLSSuite struct {
+	BaseTLSSuite
+}
+
+func (s *TLSSuite) TestTLSflagWithSelfSignedCertificate() {
+	_, out, err := s.ExecuteTestCobraCommand("list", "--tls",
+		"--api-host", s.Host,
+		"--api-port", fmt.Sprint(s.Port))
```

Comment:
You no longer need to provide these as they'll be extracted from the suite by the ExecuteTestCobraCommand in that suite.